tests/system_tests_bad_configuration.py

```diff
@@ -112,11 +113,11 @@ def wait_for_unresolvable_host(self):
         with open('../setUpClass/test-router.log', 'r') as router_log:
             log_lines = router_log.read().split("\n")
             expected_errors = [
-                "proton:io Name or service not known",  # Linux
-                "proton:io unknown node or service",  # macOS
+                "Connection to %s" % self.unresolvable_host_name
```

Comment:
Such error message does not necessarily has to be caused by unresolvable host.

When I was looking into this due to the bubblewrap problem, I was thinking that maybe I should skip the test if I get a failure when contacting a DNS server (because without working DNS I can't properly test the unresolvable host scenario). In the end I was able to get the test pass without modifying it.

I guess this change is OK, I believe the changed test is able to catch pretty much the same bugs it could previously.